Amplify/DefaultPlugins/AWSUnifiedLoggingPlugin/Internal/OSLogWrapper.swift

```diff
@@ -9,7 +9,9 @@ import os.log
 
 final class OSLogWrapper: Logger {
     private let osLog: OSLog
-
+    
+    var enable: Bool = true
```

Comment:
nit: rename to `enabled` or `loggingEnabled`

Reply:
variable renamed to `enabled` as suggested